Client.xcodeproj/xcshareddata/xcschemes/L10nSnapshotTests.xcscheme

```diff
@@ -24,8 +24,8 @@
    </BuildAction>
    <TestAction
       buildConfiguration = "Fennec"
-      selectedDebuggerIdentifier = "Xcode.DebuggerFoundation.Debugger.LLDB"
```

Comment:
I'm porting the changes for FXIOS-6050 for `L10nSnapshotTests`. They seem to help the simulators to run smoothly.

See https://github.com/mozilla-mobile/firefox-ios/pull/13737 for details